src/api/java/com/github/minecraftschurlimods/arsmagicalegacy/api/util/IDataManager.java

```diff
@@ -11,26 +10,20 @@
  */
 public interface IDataManager<T> extends Map<ResourceLocation, T> {
     /**
-     * Get the value for the id or empty.
-     *
-     * @param id the id of the requested value
-     * @return an {@link Optional} of the requested value or {@link Optional#empty()} if it is not loaded
+     * @param id The id of the requested value.
+     * @return An optional of the requested value, or an empty optional if it is not loaded.
      */
     Optional<T> getOptional(ResourceLocation id);
 
     /**
-     * Get the value for the id or throw.
-     *
-     * @param id the id of the requested value
-     * @return the requested value
+     * @param id The id of the requested value.
+     * @return The requested value. Throws an exception if it is not loaded.
      */
     T get(ResourceLocation id);
 
     /**
-     * Get the value for the id or null.
-     *
-     * @param id the id of the requested value
-     * @return the requested value or null if it is not loaded
+     * @param id The id of the requested value.
+     * @return An optional of the requested value, or null if it is not loaded.
      */
-    @Nullable T getNullable(ResourceLocation id);
+    T getNullable(ResourceLocation id);
```

Comment:
Why remove the `@Nullable` from a nullable method in a package that is `@MethodsReturnNonNullByDefault`?

Reply:
Readded it.